newapi/ooniapi/cli/__init__.py

```diff
@@ -27,6 +27,7 @@ def shell():
         code.interact(banner="", local=current_app.make_shell_context())
 
 
+# FIXME unused / dead
```

Comment:
I suggest removing it in this PR, since you have removed below `oonidata.database.Base`, hence calling this will result in an import error

Reply:
Done